lib/dom.js

```diff
@@ -751,6 +752,20 @@ declare class HTMLFormElement extends HTMLElement {
     submit(): void;
 }
 
+declare class HTMLIFrameElement extends HTMLElement {
+    allowFullScreen: boolean;
+    frameBorder: number;
+    height: number;
+    marginHeight: number;
+    marginWidth: number;
+    name: string;
+    scrolling: string;
+    sandbox: string;
```

Comment:
yeah I think it's its own class, it has methods like `toggle` and `support` so it's definitely not just a string-compatible object. this seems pretty edge-casey so if you feel like adding it then go for it, otherwise I'd just leave a stub like

```
class DOMTokenList {
  // TODO: https://dom.spec.whatwg.org/#interface-domtokenlist
}
```